test/test-listing-fei.spec.ts

```diff
@@ -0,0 +1,346 @@
+      LTV: '0',
+    });
+
+    // preparing for tests.
+    aFei = (await ethers.getContractAt('IERC20', aTokenAddress, proposer)) as IERC20;
+    stableDebt = (await ethers.getContractAt('IERC20', stableDebtTokenAddress, proposer)) as IERC20;
+    variableDebt = (await ethers.getContractAt(
+      'IERC20',
+      variableDebtTokenAddress,
+      proposer
+    )) as IERC20;
+    await (await fei.connect(feiHolder).approve(pool.address, parseEther('200000'))).wait();
+    await (await aave.connect(proposer).approve(pool.address, parseEther('200000'))).wait();
+
+    // AAVE deposit by proposer
+    await (await pool.deposit(aave.address, parseEther('100'), proposer.address, 0)).wait();
+    // FEI deposit by ampl holder
+    const depositedAmount = parseEther('100').div(decimalMultiplier);
+    await (
+      await pool.connect(feiHolder).deposit(fei.address, depositedAmount, FEI_HOLDER, 0)
+    ).wait();
+    expect(await aFei.balanceOf(FEI_HOLDER)).to.gte(depositedAmount.sub(1));
+    expect(await aFei.balanceOf(FEI_HOLDER)).to.lte(depositedAmount.add(1));
+
+    // FEI holder not able to borrow DAI against FEI
+    await expect(
+      pool.connect(feiHolder).borrow(dai.address, parseEther('1'), 2, 0, FEI_HOLDER)
+    ).to.be.revertedWith(ERRORS.NO_COLLATERAL_BALANCE);
+
+    // proposer able to borrow FEI variable against AAVE
+    const borrowAmount = parseEther('10').div(decimalMultiplier);
+    await (
+      await pool.connect(proposer).borrow(fei.address, borrowAmount, 2, 0, proposer.address)
+    ).wait();
+    expect(await variableDebt.balanceOf(proposer.address)).to.be.equal(borrowAmount);
+
+    // proposer not able to borrow FEI stable against AAVE
+    await expect(
+      pool.borrow(fei.address, borrowAmount, 1, 0, proposer.address)
+    ).to.be.revertedWith(ERRORS.NO_STABLE_BORROW);
+    increaseTime(40000);
+
+    // proposer able to repay FEI variable
+    await (await fei.connect(proposer).approve(pool.address, parseEther('100000'))).wait();
+    await (await pool.repay(fei.address, MAX_UINT_AMOUNT, 2, proposer.address)).wait();
+    expect(await variableDebt.balanceOf(proposer.address)).to.be.equal(parseEther('0'));
+  });
+
+  it("Incentives active on borrowing", async () => {
+    // The admin + emissions manager can't be the same address due to proxy restrictions 
+    // so we change it
+    await (await incentivesController.connect(feiDao).changeAdmin(FEI_HOLDER)).wait();
+    
+    await (await pool.deposit(aave.address, parseEther('100'), proposer.address, 0)).wait();
+
+        // proposer able to borrow FEI variable against AAVE
+    const borrowAmount = parseEther('10').div(decimalMultiplier);
+        await (
+          await pool.connect(proposer).borrow(fei.address, borrowAmount, 2, 0, proposer.address)
+        ).wait();
+    
+        await increaseTime(86400);
+
+    await (
+      await incentivesController
+        .connect(feiDao)
+        .configureAssets([VARIABLE_DEBT_TOKEN], [parseEther('100')])
```

Comment:
You need to get the real `variableDebtToken` address. The `VARIABLE_DEBT_TOKEN` gets placed under a proxy when the asset is listed. Also, set an smaller emission or you will cause an index overflow.
```
    const feiReserveData = await pool.getReserveData(fei.address);
    const emission = parseEther('0.00002');
    await (
      await incentivesController
        .connect(feiDao)
        .configureAssets([feiReserveData.variableDebtTokenAddress], [emission])
    ).wait();
 ```

Reply:
By real do you mean the address of the proxy or the implementation?

Reply:
proxy address